index.d.ts

```diff
@@ -459,6 +459,11 @@ export interface Player {
   gamemode: number
   ping: number
   entity: Entity
+  properties: Array<{
```

Comment:
Can you add to the doc what kind of information is here?

Reply:
You mean this? https://wiki.vg/Mojang_API#UUID_to_Profile_and_Skin.2FCape

Reply:
Yes, mineflayer should not be returning arbitrary server that can change and break user code. Should be abstracted as much as possible

Reply:
@extremeheat maybe you didnt see my message here?

Reply:
What would example usage of this PR look like in its current form? What information can you extract from this? If it's skin data, you add a `skin` object to the player object, for example.

Reply:
I currently use the data in the following form:

```typescript
function extractSkinInformation(player: mineflayer.Player & any): {url: string; model: string} | null {
  if (!player.properties) {
    return null;
  }

  const skinProperty = player.properties.find((property: any) => property.name === 'textures');
  if (!skinProperty) {
    return null;
  }

  const skinTexture = JSON.parse(Buffer.from(skinProperty.value, 'base64').toString('utf8'));

  const skinTextureHash = skinTexture?.textures?.SKIN?.url ?? null;
  const skinTextureModel = skinTexture?.textures?.SKIN?.metadata?.model ?? null;

  if (!skinTextureHash) {
    return null;
  }

  return { url: skinTextureHash, model: skinTextureModel };
}
```

But I find it hard to anticipate any future usage patterns. Even more because Mojang designed the `properties` object in a way that allows further expansion by them. I dont really think that implementing a "skin parser" would be very future proof, it would just add maintenance burden for this project.

Reply:
> find it hard to anticipate any future usage patterns

This is exactly why we do abstraction (the point of mineflayer over nmp). Likewise, we don't just emit plain packet structures back through mineflayer because they can break functionality from one version to the next. 

If there is a specific field field you're looking to use, that should be exposed, put in the doc and ideally tested (if possible). If it breaks in a future version, our tests should capture it and it can be handled internally in mineflayer to avoid a breaking change for the user.

Reply:
That function you provided, and its output, seems like what should be in and returned by mineflayer, for example:

```ts
player.skinData = { textureURL: '', model: '' } as { textureURL: string, model: string }
```